pkg/sql/create_table.go

```diff
@@ -2469,14 +2473,15 @@ func CreateRowLevelTTLScheduledJob(
 	s jobs.ScheduledJobStorage,
 	owner username.SQLUsername,
 	tblDesc *tabledesc.Mutable,
+	rowLevelTTL *catpb.RowLevelTTL,
 ) (*jobs.ScheduledJob, error) {
-	if !tblDesc.HasRowLevelTTL() {
+	if rowLevelTTL == nil {
 		return nil, errors.AssertionFailedf("CreateRowLevelTTLScheduledJob called with no .RowLevelTTL: %#v", tblDesc)
```

Comment:
nit: Update this error message to match the change to the if statement

Reply:
I undid this change.